src/app/nav/navbar.component.ts

```diff
@@ -9,21 +9,21 @@ import { AuthService } from "../user/shared/authentication.service";
   styleUrls: ['navbar.component.css']
 })
 
-export class NavbarComponent implements OnInit {
+export class NavbarComponent implements OnInit, OnDestroy {
   isLoggedIn= this.auth.isLoggedIn$.pipe(take(1));
```

Comment:
When you're binding an observable to the view using ` | async`, angular is going to handle the unsubscribing. This means you don't really need to do the take 1 here. This means it will only check once, so if your login state changes while on this page you won't be alerted of the change on the view. If you remove the take 1, the html will react to you no longer being logged in.